InventoryInStorePickupSales/Model/Order/AddStorePickupAttributesToOrder.php

```diff
@@ -0,0 +1,95 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+
+namespace Magento\InventoryInStorePickupSales\Model\Order;
+
+use Magento\InventoryInStorePickupSalesApi\Model\IsStorePickupOrderInterface;
+use Magento\Sales\Api\Data\OrderInterface;
+use Magento\Framework\Stdlib\DateTime\TimezoneInterface;
+use Magento\Sales\Api\OrderRepositoryInterface;
+use Magento\Sales\Model\Order;
+
+/**
+ * Modify order attributes according to store pickup information.
```

Comment:
Would be nice if comment could be more precise on what is modified. Still have to read code to find out what is this service doing.